en/deploy-on-azure-aks.md

```diff
@@ -0,0 +1,579 @@
+
+# create tidb node pool, the suggested nodeType is Standard_F8s_v2 or higher, by default only two TiDB nodes are required, so you can set the `--node-count` of the `tidb` node pool to `2`. You can scale out this node pool any time if necessary
+az aks nodepool add --name tidb \
+    --cluster-name ${clusterName} \
+    --resource-group ${resourceGroup} \
+    --node-vm-size ${nodeType} \
+    --zones 1 2 3 \
+    --aks-custom-headers EnableAzureDiskFileCSIDriver=true \
+    --node-count 2 \
+    --labels dedicated=tidb \
+    --node-taints dedicated=tidb:NoSchedule
+
+# create tikv node pool, the suggested nodeType is Standard_E8s_v4 or higher
+az aks nodepool add --name tikv \
+    --cluster-name ${clusterName} \
+    --resource-group ${resourceGroup} \
+    --node-vm-size ${nodeType} \
+    --zones 1 2 3 \
+    --aks-custom-headers EnableAzureDiskFileCSIDriver=true \
+    --node-count 3 \
+    --labels dedicated=tikv \
+    --node-taints dedicated=tikv:NoSchedule \
+    --enable-ultra-ssd
+```
+
+### deploy component node pools in limited zone
+
+Azure AKS cluster deploy nodes across multiple zones using "best effort zone balance", if you need "strict zone balance" (not supported in AKS now), consider deploy one node pool in each zone. For example:
+
+{{< copyable "shell-regular" >}}
+
+``` shell
+# create tikv node pool1 in zone 1
+az aks nodepool add --name tikv1 \
+    --cluster-name ${clusterName} \
+    --resource-group ${resourceGroup} \
+    --node-vm-size ${nodeType} \
+    --zones 1 \
+    --aks-custom-headers EnableAzureDiskFileCSIDriver=true \
+    --node-count 1 \
+    --labels dedicated=tikv \
+    --node-taints dedicated=tikv:NoSchedule \
+    --enable-ultra-ssd
+
+# create tikv node pool2 in zone 2
+az aks nodepool add --name tikv2 \
+    --cluster-name ${clusterName} \
+    --resource-group ${resourceGroup} \
+    --node-vm-size ${nodeType} \
+    --zones 2 \
+    --aks-custom-headers EnableAzureDiskFileCSIDriver=true \
+    --node-count 1 \
+    --labels dedicated=tikv \
+    --node-taints dedicated=tikv:NoSchedule \
+    --enable-ultra-ssd
+
+# create tikv node pool3 in zone 3
+az aks nodepool add --name tikv3 \
+    --cluster-name ${clusterName} \
+    --resource-group ${resourceGroup} \
+    --node-vm-size ${nodeType} \
+    --zones 3 \
+    --aks-custom-headers EnableAzureDiskFileCSIDriver=true \
+    --node-count 1 \
+    --labels dedicated=tikv \
+    --node-taints dedicated=tikv:NoSchedule \
+    --enable-ultra-ssd
+```
+
+> **Warning:**
+>
+> About node pool scale in:
+>
+> * You can manually scale an AKS cluster to run a different number of nodes. When you scale down, nodes are carefully [cordoned and drained](https://kubernetes.io/docs/tasks/administer-cluster/safely-drain-node/) to minimize disruption to running applications, refer [Scale the node count in an Azure Kubernetes Service (AKS) cluster](https://docs.microsoft.com/en-us/azure/aks/scale-cluster).
+
+## 配置 StorageClass
```

Comment:
```suggestion
## StorageClass
```

Reply:
Thank you, have fixed the description in EN doc